src/snowflake/snowpark/_internal/analyzer/select_statement.py

```diff
@@ -1367,7 +1367,7 @@ def limit(self, n: int, *, offset: int = 0) -> "SelectStatement":
         else:
             new = copy(self)
             new.from_ = self.from_.to_subqueryable()
```

Comment:
Can we add a test case to the existing integ suite (not just simplifier suite)?

I.e., there's a good chunk of testing for _show_string already existing, would be great to built upon that!